authorization/pb/events.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+
+package ocf.cloud.auth.pb;
+
+option go_package = "github.com/plgd-dev/cloud/authorization/pb;pb";
+
+// devices was registered to owner.
+message RegisteredDevices {
```

Comment:
The event described what has happened.
`DevicesRegistered`and `DevicesUnregistered`.